src/System.Transactions.Local/src/System/Transactions/TransactionsEtwProvider.cs

```diff
@@ -4,7 +4,6 @@
 
 using System;
 using System.Collections.Generic;
-//using System.Linq;
```

Comment:
When scanning for callsites that are not at risk of `Linq` resolution, this causes a false-positive

Reply:
We can probably remove the commented out using directive, right?
cc @jimcarley, @qizhanMS

Reply:
Yes, I removed it already

Reply:
Just to close the loop - yes, removing that comment is fine.

Reply:
Thanks Jim